projects/core/src/util/normalize-http-error.ts

```diff
@@ -13,6 +13,10 @@ import { HttpErrorModel } from '../model/misc.model';
 export function normalizeHttpError(
   error: HttpErrorResponse | any
 ): HttpErrorModel | undefined {
+  if (error instanceof HttpErrorModel) {
```

Comment:
reason for this is we need throw an error in the backoff, but backoff error is already normalized.

We can't just return NEVER instead of return throwError(stuff) as it will not get caught in the subscription error of said function that failed.

Reply:
Discussed offline, good one.